Feature: Add an option to Edit menu to Manage Payees for selected transactions.

StephenBrown2 opened the pull request:
GitHub Issue (if applicable): #1644

Trello Link (if applicable):

Forum Link (if applicable):

#### Explanation of Bugfix/Feature/Modification:

This currently is a very hacky way of opening the Manage Payees dialog from a set of selected transactions using the Edit menu rather than the main account menu, which when combined with [Show Menu When Right Clicking On Transaction](https://github.com/toolkit-for-ynab/toolkit-for-ynab/blob/master/docs/feature-list.md#show-menu-when-right-clicking-on-transaction), allows for easy recategorization of transaction payees.

I just copied the "Clear Selection" feature and modified it for these purposes, so I'm sure there's a better way to do it. I'm very unfamiliar with the inner workings of Ember, so please don't merge this until we figure out the "right" way to launch the modal.

joshmadewell commented on src/extension/features/accounts/manage-payees/index.js:
```diff
@@ -0,0 +1,37 @@
+            </button>
+          </li>
+          <li><hr /><li>`).click(() => {
+        $('#ember6765').click();
```
unfortunately, it's not safe to always use this ID. if YNAB has an update this ID is likely to change.

The safer way to do this will be to just emit the action to open the modal:

```
controllerLookup('accounts').send('openPayeeModal')
```

joshmadewell commented on src/extension/features/accounts/manage-payees/index.js:
```diff
@@ -0,0 +1,37 @@
+            </button>
+          </li>
+          <li><hr /><li>`).click(() => {
+        $('#ember6765').click();
```
This ID is likely to change as soon as YNAB makes an update. We can use the accounts controller to open the modal safely:

```
controllerLookup('accounts').send('openPayeeModal')
```

StephenBrown2 replied:
Excellent! I wasn't sure how to get that. I'll update it to be sure.

joshmadewell commented on src/extension/features/accounts/manage-payees/index.js:
```diff
@@ -0,0 +1,37 @@
+    }
+  };
+
+  invoke = () => {
```
prefer to see `observe()` and `invoke()` as declarations instead of expressions -- if you wouldn't mind updating these :D

StephenBrown2 replied:
Sure thing. Directly copied from [clear-selection](https://github.com/toolkit-for-ynab/toolkit-for-ynab/blob/master/src/extension/features/accounts/clear-selection/index.js#L28), will update.

StephenBrown2 commented:
I've updated and rebased on master for those updates, and it's looking alright.

I also updated the insertion location to be before the "Move to Account ➡️" option:
<img width="289" alt="Screen Shot 2019-09-23 at 9 43 00 AM" src="https://user-images.githubusercontent.com/1148665/65441015-e5417680-dde6-11e9-94e3-c78e9a9358d1.png">

dbaldon commented:
Should the menu item say 'Manage Selected Payees'?

StephenBrown2 commented:
It could very well say that, though I believe all the items in that list are implied to be for the selected transactions.

joshmadewell commented:
I agree with Stephen, the naming for the others doesn't specify "selected" since that's implied in the context of the modal itself. the only thing I was going to suggest was to pluralize "Payees" only when necessary but we don't have a true localization tool which could handle plurality so I wasn't worried about it.

dbaldon commented:
No worries mates. Ship it!

esseti commented:
Can someone explain how to use it?
i can select various entries, right click, show payees, then select the one i want to use for all the selected entries, click done.. nothing happens.
How can I assign a payee to multiple transactions?
